app-admin/lact/spec

```diff
@@ -0,0 +1,3 @@
+VER=0.5.4
+SRCS="git::commit=tags/v$VER::https://github.com/ilya-zlobintsev/LACT"
+CHKSUMS="SKIP"
```

Comment:
Missing CHKUPDATE.

Reply:
It is not traced by anyita, now trying to using github api

Reply:
Updated